cluster-provision/k8s/1.24/manifests/cni.diff

```diff
@@ -0,0 +1,100 @@
++              "assign_ipv4": "true",
++              "assign_ipv6": "true"
++          },
++          "container_settings": {
++              "allow_ip_forwarding": true
+           },
+           "policy": {
+               "type": "k8s"
+@@ -3533,7 +3538,7 @@ spec:
+         # It can be deleted if this is a fresh installation, or if you have already
+         # upgraded to use calico-ipam.
+         - name: upgrade-ipam
+-          image: docker.io/calico/cni:v3.18.0
++          image: quay.io/calico/cni:v3.18.0
+           command: ["/opt/cni/bin/calico-ipam", "-upgrade"]
+           envFrom:
+           - configMapRef:
+@@ -3560,7 +3565,7 @@ spec:
+         # This container installs the CNI binaries
+         # and CNI network config file on each node.
+         - name: install-cni
+-          image: docker.io/calico/cni:v3.18.0
++          image: quay.io/calico/cni:v3.18.0
+           command: ["/opt/cni/bin/install"]
+           envFrom:
+           - configMapRef:
+@@ -3601,7 +3606,7 @@ spec:
+         # Adds a Flex Volume Driver that creates a per-pod Unix Domain Socket to allow Dikastes
+         # to communicate with Felix over the Policy Sync API.
+         - name: flexvol-driver
+-          image: docker.io/calico/pod2daemon-flexvol:v3.18.0
++          image: quay.io/calico/pod2daemon-flexvol:v3.18.0
+           volumeMounts:
+           - name: flexvol-driver-host
+             mountPath: /host/driver
+@@ -3612,7 +3617,7 @@ spec:
+         # container programs network policy and routes on each
+         # host.
+         - name: calico-node
+-          image: docker.io/calico/node:v3.18.0
++          image: quay.io/calico/node:v3.18.0
+           envFrom:
+           - configMapRef:
+               # Allow KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT to be overridden for eBPF mode.
+@@ -3671,6 +3676,8 @@ spec:
+             # no effect. This should fall within `--cluster-cidr`.
+             # - name: CALICO_IPV4POOL_CIDR
+             #   value: "192.168.0.0/16"
++            - name: IP6
++              value: "autodetect"
+             # Disable file logging so `kubectl logs` works.
+             - name: CALICO_DISABLE_FILE_LOGGING
+               value: "true"
+@@ -3679,12 +3686,14 @@ spec:
+               value: "ACCEPT"
+             # Disable IPv6 on Kubernetes.
+             - name: FELIX_IPV6SUPPORT
+-              value: "false"
++              value: "true"
+             # Set Felix logging to "info"
+             - name: FELIX_LOGSEVERITYSCREEN
+               value: "info"
+             - name: FELIX_HEALTHENABLED
+               value: "true"
++            - name: CALICO_IPV6POOL_NAT_OUTGOING
++              value: "true"
+           securityContext:
+             privileged: true
+           resources:
+@@ -3818,6 +3818,8 @@ spec:
+           operator: Exists
+         - key: node-role.kubernetes.io/master
+           effect: NoSchedule
++        - key: node-role.kubernetes.io/control-plane
```

Comment:
Thanks for the fix
btw, maybe latest calico has this already built in ? 
but we can bump on other PR if needed and try